### ISSUE TITLE
IBD to PTN000###### for annotation ####### to term GO:000#### does not have associated experimental evidence

### ISSUE THREAD

**Issue by dustine32** [pantherdb/fullgo_paint_update#23]
Sorry if this issue description is also cryptic. It's another example of a warning from Anushya's web service on book PTHR12651:
> IBD to PTN000291528 for annotation 313851835 to term GO:0008540 does not have associated experimental evidence

It seems related to the problem in https://github.com/pantherdb/fullgo_paint_update/issues/21. From what I can see in the DB this paint annotation does have an IDA so I suspect this warning is misleading. Perhaps @mugitty can take a look?

**Comment by mugitty** [comment]
The error is due to the qualifier.  The IBD annotation does not have a qualifier.  However, the experimental evidence associated with the annotation has qualifier 'colocalizes with'. 
The system that checks for evidence to support IBD annotations also checks for matching qualifiers.

**Comment by pgaudet** [comment]
1. Is this really an error ? 
2. Can the message be made less cryptic, for example: "IBD to PTN000291528 for annotation 313851835 to term GO:0008540 _does not have the same qualifier as experimental evidence_"? 

Thanks, Pascale

**Comment by mugitty** [comment]
The functionality that validates the IBD annotations calculates the list of possible annotations to support the IBD annotation.   Whatever does not fall in this list is rejected with the same message.  There is no handling to check for exact reason.

**Comment by pgaudet** [comment]
I wonder if this should be checked ? The rules allow us to annotate without propagating the qualifier - I dont think you should take the qualifier into account when doing those checks. 

Thanks, Pascale

**Comment by pgaudet** [comment]
Maybe @marcfeuermann @huaiyumi @thomaspd want to comment ?

**Comment by dustine32** [comment]
Yep, the qualifier missing would mean it doesn't match. I didn't notice that before. Thanks!

@mugitty Do you have any ideas of a likely culprit for this difference between the supporting GO annotation and the paint annotation? Could the GO annotation at some point have gained the qualifier in the source GAFs? Would paint qualifier creation be something that the update process should be handling or should these paint annotations be obsoleted and new annotations with their own qualifier records created?

**Comment by dustine32** [comment]
Sorry @pgaudet I just saw your qualifier propagating comment. That's a fair point about keeping the rules consistent.

**Comment by pgaudet** [comment]
@dustine32 @mugitty 
We cannot take into account the contributes_to and colocalizes_with qualifiers (but we need to take NOT into account). Many times it's inconsistent even in the EXP data - some groups will have 'contributes_to, some won't, for the same annotation.

**Comment by mugitty** [comment]
@pgaudet should the PAINT tool permit curators to create IBD annotations with contributes_to and colocalizes_with qualifiers based on 1 or more experimental annotations having contributes_to or colocalizes_with qualifier? If yes, there exists the possibility of losing the experimental annotation with qualifier due to pruning, loss of experimental data, etc.  This will result in having IBD annotations with a qualifier that does not match the qualifier of the experimental annotations  supporting the IBD annotation.

**Comment by pgaudet** [comment]
Hi @mugitty 

I do think we should ignore the contributes_to and colocalizes_with qualifiers. 

We should be allowed to make annotations with or without them, regardless of whether the original annotation has it. 

(perhaps we can skype to clarify this).

Thanks, Pascale

**Comment by pgaudet** [comment]
My concern is that as far as I know, this was not checked by the older versions of PAINT. The curators were always able to choose to add a qualifier or not. Could we be getting errors that may be to adding this new check more recently ?
